### ISSUE TITLE
Items of itemize environment should have "ListItem" types in the textlint AST

### ISSUE THREAD

**Issue by kn1cht** [textlint/textlint-plugin-latex2e#53]
## Problem

[textlint-rule-ja-no-mixed-period](https://github.com/textlint-ja/textlint-rule-ja-no-mixed-period/)  ignores lists and doesn't throw error even if list items don't have periods.

https://github.com/textlint-ja/textlint-rule-ja-no-mixed-period/blob/3c9eb6f5b7b94a557a410e3367cffc90b883115b/src/textlint-rule-ja-no-mixed-period.js#L51-L57

However, `\item` without periods in LaTeX files cause error of ja-no-mixed-period.

![image](https://user-images.githubusercontent.com/16277200/103517861-282d3d80-4eb6-11eb-8f33-236ae390fba4.png)

## Cause
```tex
\\begin{document}
  \\begin{itemize}
    \\item item1
     \\item item2
  \\end{itemize}
\\end{document}
```

Currently, plugin-latex2e converts this LaTeX code to an AST like:

```json
{
  "type": "Document",
  "raw": "...",
  "range": [],
  "loc": {},
  "children": [
    {
      "loc": {},
      "range": [],
      "raw": "item1\n      \\item item2",
      "type": "Paragraph",
      "children": [
        {
          "loc": {},
          "range": [],
          "raw": "item1",
          "type": "Str",
          "value": "item1"
        },
        {
          "loc": {},
          "range": [],
          "raw": "\n      \\item ",
          "type": "Html",
          "value": "\n      \\item "
        },
        {
          "loc": {},
          "range": [],
          "raw": "item2",
          "type": "Str",
          "value": "item2"
        }
      ]
    }
  ]
}
```

## Expected Behavior
![image](https://user-images.githubusercontent.com/16277200/103518754-ab9b5e80-4eb7-11eb-8f9a-554a1676e0ef.png)

textlint AST sets "ListItem" types to items of lists.
plugin-latex2e should also set "ListItem" to items of itemize environment.
